build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -343,6 +343,18 @@ class SeleniumWebDriverController {
     fs.writeFile(path, imageString, 'base64', function() {});
   }
 
+  /**
+   * Evaluate the given function
+   * @param {function()} fn
+   * @param {...*} unusedArgs
+   * @return {!ControllerPromise}
+   * @override
+   */
+  eval_(fn, unusedArgs) {
+    const args = Array.prototype.slice.call(arguments, 1);
```

Comment:
A bit unclear here, `[fn].concat(args)` looks to be the same thing as arguments, so why not just apply executeScript with arguments directly?

If you do want to do this pattern (e.g. if typing is an issue), this would be better:

```javascript
eval_(fn, ...args) {
  return this.driver.executeScript(fn, ...args);
}
```

Reply:
I didn't want to use `arguments` directly because I wanted to enforce the use of a function as the first argument, since I believe Selenium's `executeScript` also accepts a string. The spread operator works better, thanks